{"name": "Проверка невозможности авторизации курьера при некорректных данных", "status": "passed", "start": 1732976693204, "stop": 1732976693325, "uuid": "9375df17-2ef0-4fee-b3b0-3982ce27612c", "historyId": "dab1b80a6c3ce035d28f86e9b1b2b882", "testCaseId": "dab1b80a6c3ce035d28f86e9b1b2b882", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_courier_uncorrect_login_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-4GFTOQP"}, {"name": "thread", "value": "10124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}